src/Core/Logger/Factory/LegacyLoggerFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+// Copyright (C) 2010-2024, the Friendica project
+// SPDX-FileCopyrightText: 2010-2024 the Friendica project
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+declare(strict_types=1);
+
+namespace Friendica\Core\Logger\Factory;
+
+use Friendica\Core\Config\Capability\IManageConfigValues;
+use Friendica\Core\Hooks\Capability\ICanCreateInstances;
+use Friendica\Util\Profiler;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Manager for the core logging instances
```

Comment:
This is the same description as the `LoggerManager` class, can you please expand on the `Legacy` part and what need to be done to eventually remove it?

Reply:
I've updated the description.